device-connectors/src/testflinger_device_connectors/devices/__init__.py

```diff
@@ -48,6 +47,7 @@
     "noprovision",
     "oemrecovery",
     "oemscript",
+    "zapper_iot",
```

Comment:
It looks like this is only meant as a base class for other device connectors to use, correct? If so, then you don't want to put it in DEVICE_CONNECTORS. These are intended for things that will be used by real devices and not just inherited, and will turn into subcommands for testflinger-device-connector cli

Reply:
The base class is just `zapper` and it's indeed not included in that list. On the other hand, `zapper_iot` would be the first real connector to inherit from it and it's the one owned by the OEM team. If you prefer, we could add it in there only once it gets properly implemented, for now I was just trying to set the ground for them.

Reply:
Ah, right. No I think that's fine then to go ahead and add it, since that's going to be the actual device agent that implements it. +1